assets/schema_input.json

```diff
@@ -1,6 +1,6 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "$id": "https://raw.githubusercontent.com/ebi-metagenomics/miassembler/master/assets/schema_input.json",
+    "$id": "https://raw.githubusercontent.com/ebi-metagenomics/emg-viral-pipeline/dev/assets/schema_input.json",
```

Comment:
We need to remember to update this `dev` -> `master`

Reply:
Yes, absolutely (bit now dev if quite different from master)